src/manifest.rs

```diff
@@ -136,3 +143,18 @@ pub fn write_package_json(
 pub fn get_crate_name(path: &str) -> Result<String, Error> {
     Ok(read_cargo_toml(path)?.package.name)
 }
+
+fn has_cdylib(path: &str) -> Result<bool, Error> {
+    Ok(read_cargo_toml(path)?.lib.map_or(false, |lib| {
+        lib.crate_type
+            .map_or(false, |types| types.iter().any(|s| s == "cdylib"))
+    }))
+}
+
+pub fn check_crate_type(path: &str) -> Result<(), Error> {
+    if !has_cdylib(path)? {
+        Error::config("crate-type must include cdylib to compile to wasm32-unknown-unknown")
```

Comment:
it would be awesome if we can give the exact code a user would need to put into their cargo.toml to fix the issue